test/__snapshots__/main.test.ts.snap

```diff
@@ -1874,8 +1874,8 @@ export default function getRoutes(config: RouteConfig) {
 
         },
         data: {
-          availableVehicleId,
-          specialEquipments
+          availableVehicleId: JSON.stringify(availableVehicleId),
```

Comment:
For example, here the expected encoding would be:

```typescript
availableVehicleId: m.UUID.encode(availableVehicleId),
specialEquipments: Array<m.SelectedSpecialEquipment>.encode(specialEquipments)
```

This makes the serialization 100% consistent with the deserialization in case of custom io-ts types